snuba/subscriptions/scheduler_processing_strategy.py

```diff
@@ -454,7 +454,7 @@ def submit(self, message: Message[CommittableTick]) -> None:
         # to commit is provided.
         if len(encoded_tasks) == 0 and message.payload.offset_to_commit is not None:
             offset = {message.value.partition: message.value.payload.offset_to_commit}
-            logger.info("Committing offset: %r", offset)
+            logger.info("Committing offset - no subscriptions: %r", offset)
             self.__commit(offset)
```

Comment:
can you add `force=True` in join() of this strategy?

Reply:
Good point, will update

Reply:
there's generally no guarantee of exactly-once so I am not sure if it is worth restarting CI over rn though

Reply:
race condition with you posting comment... already pushed